### PR TITLE
Add loki-traefik relation

### PR DESCRIPTION
**Pull request by sed-i**
## Issue
Load test deploys the bundle, but it does not include the recently added ingress for loki.

## Solution
Add loki-traefik relation.

### CODE REVIEW

File: bundle.yaml.j2 [hunk 1]
```diff
@@ -72,7 +72,8 @@ applications:
   {%- endif %}
 
 relations:
-- [prometheus:ingress, traefik]
+- [traefik, prometheus:ingress]
+- [traefik, loki:ingress]
```
Comment (Abuelodelanada) [inline bundle.yaml.j2:76]: I would make explicit both sides of the relation just  as the other relations we have in the file

**Review by Abuelodelanada — APPROVED** [review]

File: tox.ini [hunk 2]
```diff
@@ -67,8 +69,8 @@ commands =
 description = Run integration tests
 deps =
     jinja2
-    git+https://github.com/juju/python-libjuju.git
-    #juju
+    #git+https://github.com/juju/python-libjuju.git
```
Comment (dstathis) [inline tox.ini:72]: I would just delete this comment.

**Review by dstathis — APPROVED** [review]

File: tox.ini (hunk 2 above)
Comment (simskij) [reply tox.ini:72]: Agreed

Comment (sed-i) [reply tox.ini]: Done. We do toggle them quite often though, don't we?

**Pull request merged**